### DISCUSSION TITLE
Waveform not working before updated to 2021.1127.0

### DISCUSSION THREAD

**Discussion by Akarinnnnn** [ppy/osu#15833] [Q&A] [answered]
Logs has been override.....

**Comment by bdach** [comment, marked as answer]
whatever it is you are trying to report, i do not understand it. no screenshots, no logs, no anything, just a vague one-sentence claim.

if you are going to open discussions like this then please just don't and save yourself (and us) the trouble.

**Reply by Akarinnnnn** [reply]
Sorry for that, I can't delete or hide this myself since I realized this discussion is helpless, could you help me to remove it?
